events/sick_man_events.txt

```diff
@@ -0,0 +1,664 @@
+					NOT = {
+						any_overlord_or_above = {
+							NOT = {
+								ROOT = this
+							}
+						}
+					}
+					NOT = {
+						is_country_type = unrecognized
+					}
+				}
+			}
+			every_subject_or_below = {
+				limit = {
+					NOT = {
+						any_overlord_or_above = {
+							NOT = {
+								ROOT = this
+							}
+						}
+					}
+					OR = {
+						NOT = {
+							is_country_type = unrecognized
+						}
+						is_subject_type = subject_type_protectorate
+						is_subject_type = subject_type_dominion
+						is_subject_type = subject_type_puppet
+					}
+				}
+				make_independent = yes
+			}
+		}
+		add_modifier = {
+			name = dead_man_of_europe
+		}
+		set_country_type = unrecognized
+		ruler = {
+			kill_character = yes
+		}
+	}
+}
+
+# Choice Event
+sick_man.4 = {
+	type = country_event
+	placement = root
+	title = sick_man.4.t
+	desc = sick_man.4.d
+	flavor = sick_man.4.f
+	duration = 3
+	event_image = {
+		video = "unspecific_factory_closed"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	trigger = {
+		var:sick_man_var >= 4
+		NOR = {
+			has_variable = liberal_tanzimat
+			has_variable = absolute_tanzimat
+		   }
+	}
+	option = {
+		name = sick_man.4.a
+		set_variable = liberal_tanzimat
+		add_journal_entry = {
+			type = je_sick_man_liberal_kurdish
+		}
+		add_journal_entry = {
+			type = je_sick_man_separatism
+		}
+		add_journal_entry = {
+			type = je_sick_man_education
+		}
+		add_journal_entry = {
+			type = je_sick_man_liberal_voting
+		}
+		add_journal_entry = {
+			type = je_sick_man_liberal_secularism
+		}
+		add_journal_entry = {
+			type = je_sick_man_liberal_labor
+		}
+	}
+	option = {
+		name = sick_man.4.b
+		set_variable = absolute_tanzimat
+		add_journal_entry = {
+			type = je_sick_man_absolute_kurdish
+		}
+		add_journal_entry = {
+			type = je_sick_man_education
+		}
+		add_journal_entry = {
+			type = je_sick_man_separatism
+		}
+		add_journal_entry = {
+			type = je_sick_man_syria
+		}
+		add_journal_entry = {
+			type = je_sick_man_egypt
+		}
+		add_journal_entry = {
+			type = je_sick_man_absolute_islam
+		}
+	}
+}
+
+# Absolute Success event
+sick_man.5 = {
+	type = country_event
+	placement = root
+	title = sick_man.5.t
+	desc = sick_man.5.desc
+	flavor = sick_man.5.f
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+	duration = 3
+	event_image = {
+		video = "europenorthamerica_london_center"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	trigger = {
+	}
+	immediate = {
+		set_variable = sick_man_complete
+		remove_modifier = sick_man_of_europe
+		remove_modifier = outmoded_bureaucracy
+		remove_modifier = cizye_tax
+		remove_modifier = stagnant_economy
+		remove_modifier = janissary_legacy
+		remove_modifier = rampant_piracy
+		remove_modifier = kurdish_question
+		remove_modifier = balkan_disarray
+		remove_modifier = imbalanced_education
+		remove_modifier = internal_strife
```

Comment:
I believe you should use ?= since the modifier can already be removed. Also I believe ?= works, but I'm not 100% sure. Otherwise it's if = { limit = { modifier_exists }} for every modifier.

Reply:
`remove_modifier ?= modifier_name` definitely works and should be used here

Reply:
what is ?=

Reply:
It basically means "if this exists, do this". In this case, it might seem redundant, to do a "if `sick_man_of_europe` exists, remove it", but Vic3 starts spitting out errors if it gets the order to remove a modifier that does not exist.